resources/zh/cemu.po

```diff
@@ -85,6 +57,122 @@ msgstr "输入内存地址"
 msgid "Memory breakpoint"
 msgstr "内存断点"
 
+#: CemuApp.cpp
+msgid "Browse"
+msgstr "浏览"
+
+#: CemuApp.cpp
+msgid "Select a file"
+msgstr "选择一个文件"
+
+#: CemuApp.cpp
+msgid "Select a directory"
+msgstr "选择一个目录"
+
+#: CemuApp.cpp
+msgid "base"
+msgstr "本体"
+
+#: CemuApp.cpp
+msgid "update"
+msgstr "更新档"
+
+#: CemuApp.cpp
+msgid "dlc"
+msgstr "DLC / 附加内容"
+
+#: CemuApp.cpp
+msgid "save"
+msgstr "存档"
+
+#: CemuApp.cpp
+msgid "Japan"
+msgstr "日本"
+
+#: CemuApp.cpp
+msgid "USA"
+msgstr "美国"
+
+#: CemuApp.cpp
+msgid "Europe"
+msgstr "欧洲"
+
+#: CemuApp.cpp
+msgid "Australia"
+msgstr "澳大利亚"
+
+#: CemuApp.cpp
+msgid "China"
+msgstr "中国"
+
+#: CemuApp.cpp
+msgid "Korea"
+msgstr "韩国"
+
+#: CemuApp.cpp
+msgid "Taiwan"
+msgstr "中国台湾"
+
+#: CemuApp.cpp
+msgid "Auto"
+msgstr "自动"
+
+#: CemuApp.cpp
+msgid "many"
+msgstr "many/多区域"
```

Comment:
This is difficult to determine. "many" is sandwiched between regions and languages in the translation file. The former is the region in the game list, so it may be more appropriate to translate into multiple regions. In order to prevent translation errors, I attached English to ensure that translation problems will not occur. If the correct translation method is determined later, I will add it.

Reply:
I will ask to the Cemu devs.

Reply:
A Cemu dev told me it refers to multi-region, so your guess is correct. Remove the original "many" the next time you edit the file.
Thanks for having made me aware of that 😀